README.md

```diff
@@ -172,13 +177,12 @@ Another key goal of having a robust Messaging layer under the covers is that the
 - `POST` a [trivial YAML/JSON payload](#yaml-to-submit-a-transaction)
   - to `/hook` to complete once the message is confirmed by Kafka (0.5s - tunable)
   - to `/fasthook` to complete immediately when the message is sent to Kafka
-- Get an _immediate_ receipt
-   - No need to wait many seconds for a block to be cut
-   - No need to have complex retry handling
+- Receive back an `id` for the request straight away
+   - Let the bridge do the retry polling to get the Ethereum receipt once a block is cut
```

Comment:
Receive a webhook once the receipt is received?

Reply:
Today we don't have way to do HTTP webhooks _back_ to the application. Instead we provide a store of transactions you query the status. That means HTTP polling.
So the trivially simple HTTP API, is just like the JSON/RPC API in that regards - you get an ID, and to know the success you have to poll.

If you want full async programming today, then that's available by coding to Kafka directly.